codebuild_specs/release_workflow.yml

```diff
@@ -0,0 +1,44 @@
+version: 0.2
+env:
+  shell: bash
+  compute-type: BUILD_GENERAL1_MEDIUM
+
+batch:
+  fast-fail: false
+  build-graph:
+    - identifier: build_linux
+      buildspec: codebuild_specs/build_linux.yml
+    - identifier: test
+      buildspec: codebuild_specs/test.yml
+      depend-on:
+        - build_linux
+    - identifier: lint
+      buildspec: codebuild_specs/lint.yml
+      depend-on:
+        - build_linux
+    - identifier: verify_api_extract
+      buildspec: codebuild_specs/verify_api_extract.yml
+      depend-on:
+        - build_linux
+    - identifier: verify_yarn_lock
+      buildspec: codebuild_specs/verify_yarn_lock.yml
+      depend-on:
+        - build_linux
+    - identifier: verify_cdk_version
+      buildspec: codebuild_specs/verify_cdk_version.yml
+      depend-on:
+        - build_linux
+    - identifier: mock_e2e_tests
+      buildspec: codebuild_specs/mock_e2e_tests.yml
+      depend-on:
+        - build_linux
+    - identifier: publish_to_local_registry
```

Comment:
If the e2e tests are not in the release workflow, the `publish_to_local_registry` can be removed